javascript/phone.js

```diff
@@ -0,0 +1,69 @@
+(function () {
+  class Phone {
+    constructor(rootE) {
+      this.rootE = rootE;
+      this.conInput = this.rootE.querySelector(".phone__container__input");
```

Comment:
Let's try to improve the naming here. Why does it include "container" in the name? It's not a container, it's an input, so we could rename it to something like phone_input